profiler/README.md

```diff
@@ -1,9 +1,46 @@
 
-The profiler is in its infancy and is disabled by default.
+🚨 The profiler is in beta and its use is currently restricted.
 
-It should be considered experimental and is completely unsupported.
+# About
 
-# configuration
+The `splunk-otel-java` agent includes an "always-on" profiler that can be enabled with a configuration
+setting. This profiler periodically captures the call stack state for all JVM threads and
+saves these to the Splunk Observability Cloud. Users can then view a flamegraph of application
+call stacks and inspect individual code-level call stacks for relevant traces.
```

Comment:
```suggestion
The `splunk-otel-java` agent includes a continuous profiler that can be enabled at any time. This profiler periodically captures the call stack state for all JVM threads and send them to Splunk Observability Cloud. You can then view a flamegraph of application call stacks and inspect individual code-level call stacks for relevant traces.
```

(DD calls that "Continuous profiler".)

Reply:
I don't mind the change around "always-on" to "continuous", no preference.

 I'm opposed to changing the wording to "enabled at any time" because it's in accurate. It has to be configured at startup time and cannot be changed in a running application, which is what the "enabled at any time" suggests.

Reply:
Suggestion updated.

Reply:
still says "at any time" for me tho.

Reply:
How about now?

Reply:
Now it says it is always enabled...which is not true. By default it is NOT enabled (yet).  :)

Reply:
Gotcha. Reworded.